drivers/docker/driver.go

```diff
@@ -23,6 +23,9 @@ import (
 	multierror "github.com/hashicorp/go-multierror"
 	plugin "github.com/hashicorp/go-plugin"
 	"github.com/hashicorp/go-set"
+	"github.com/ryanuber/go-glob"
+	"golang.org/x/exp/slices"
+
```

Comment:
extra newline in these imports?

Reply:
eh I think it's my goland goimports setting. I'll revert this.